Wrappers/Python/cil/optimisation/functions/TotalVariation.py

```diff
@@ -261,9 +272,10 @@ def proximal(self, x, tau, out = None):
         p1 = self.gradient.range_geometry().allocate(0)
 
         if self.strong_convexity_constant>0:
-            tau /= (1 + tau*self.strong_convexity_constant)
+
             x /= (1 + tau*self.strong_convexity_constant)
-        
+            tau /= 1 + tau*self.strong_convexity_constant
```

Comment:
I suppose the `/=` operator makes so that the first thing that happens here is the evaluation of `1+ tau*self.strong_convexity_constant`, but I think the following would be clearer, as in the above line.
```

Reply:
I used `tmp_tau` instead.